src/Stack/MetricsCollector.php

```diff
@@ -0,0 +1,229 @@
+<?php
+namespace Stack;
+
+class MetricsRegistry
+{
+    public function __construct($metrics = [])
+    {
+        $storage = new \Prometheus\Storage\APC();
+        $this->registry = new \Prometheus\CollectorRegistry($storage);
+        $this->register_metrics($metrics);
+    }
+
+    public function register_metrics($metrics = [])
```

Comment:
I think that in PHP, the standard scheme for naming things is camel case.

Reply:
It appears so, yes. I'll rename them.

Reply:
Done.